ts/nni_manager/training_service/kubernetes/frameworkcontroller/frameworkcontrollerTrainingService.ts

```diff
@@ -202,6 +301,10 @@ class FrameworkControllerTrainingService extends KubernetesTrainingService imple
             const fcClusterConfigNFS: FrameworkControllerClusterConfigNFS = <FrameworkControllerClusterConfigNFS>this.fcClusterConfig;
             const nfsConfig: NFSConfig = fcClusterConfigNFS.nfs;
             return `nfs://${nfsConfig.server}:${destDirectory}`;
+        } else if (this.fcClusterConfig.storage === 'pvc') {
+            await cpp.exec(`mkdir -p ${this.trialLocalNFSTempFolder}/${destDirectory}`);
```

Comment:
trialLocalNFSTempFolder -> trialLocalPVCTempFolder

Reply:
I unified it :)